### DISCUSSION TITLE
Change the width of the chat window?

### DISCUSSION THREAD

**Discussion by tusharbhutt** [oobabooga/text-generation-webui#5141] [Q&A] [answered]
Hi, is there any way to change the width of the chat window?  Right now there is a lot of wasted space on both sides, and outputting a 1024x1024 image from AUTO1111 requires scrolling. 

Is it in a JSON file, or a YAML or somewhere else that can be preset?

**Comment by TheLounger** [comment, marked as answer]
There is currently no setting or preset for this. You can change it in `css/main.css` for the time being.
It crossed my mind to have several style settings be available as user settings, but for now it requires manual editing.

**Comment by tusharbhutt** [comment]
Thanks for the info, hopefully it doesn't get overwritten with each update :)
